src/lsm/lsm_tree.c

```diff
@@ -1378,9 +1378,9 @@ __wt_lsm_tree_worker(WT_SESSION_IMPL *session,
 	u_int i;
 	bool exclusive, locked, need_release;
 
-	locked = false;
-	need_release = false;
+	locked = need_release = false;
```

Comment:
false initialisation here seems redundant because both locked and need_release are set to true a few lines below.

Reply:
@sulabhM, agreed -- I'll send out email to explain my thinking.